modules/Forum/pages/forum/get_quotes.php

```diff
@@ -20,28 +20,18 @@
 $forum = new Forum();
 
 // Get the post data
-if (empty($_POST)) {
+if (empty($_GET)) {
     die(json_encode(['error' => 'No post data']));
 }
 
-$posts = [];
+$post = $forum->getIndividualPost($_GET['post']);
 
-foreach ($_POST['posts'] as $item) {
-    $post = $forum->getIndividualPost($item);
+$content = $post['content'];
 
-    $content = $post['content'];
-    $content = preg_replace('~<blockquote(.*?)>(.*)</blockquote>~si', '', $content);
```

Comment:
This line caused issues when quoting a post which contained another quote. Removing it allows nested quotes like so:
<img width="1211" alt="Screenshot 2023-01-14 at 00 43 26" src="https://user-images.githubusercontent.com/26070412/212463990-a7ff7c4d-5180-4b60-ac79-105c0cafdb9f.png">

If there is a reason this has to stay, let me know